php/thankYou.php

```diff
@@ -105,12 +264,36 @@
         <p class="lead">Please review the following information you have submitted</p>
     </div>
 
+    <?php
+
+    //Validates agree button and background check
+    /**
+     * Validates background and agree
+     */
+    if($_POST['question'] === 'agreed'){
```

Comment:
"Notice:  Undefined index: question..." is on the page when I submit an empty form while debugging.